test/lan_test.rb

```diff
@@ -12,6 +12,10 @@
   subject { Yast::Lan }
   let(:system_config) { instance_double(Y2Network::Config, "System") }
 
+  before do
+    Yast::Lan.clear_configs
```

Comment:
maybe even hook it to general test_helper? As no test should depend on previous configs.